poem/src/middleware/tracing_mw.rs

```diff
@@ -50,16 +51,12 @@ impl<E: Endpoint> Endpoint for TracingEndpoint<E> {
                     );
                     Ok(resp)
                 }
-                (Ok(resp), Err(_)) => {
-                    let resp = resp.into_response();
+                Err(err) => {
                     tracing::info!(
-                        status = %resp.status(),
-                        "response"
+                        error = %err,
+                        duration = ?duration,
+                        "error"
                     );
```

Comment:
This is also a slight change in functionality- I personally thought it'd be useful to see the duration of failed requests too, but totally fine to change this back if you'd prefer.